packages/eslint-config-airbnb/rules/react.js

```diff
@@ -8,12 +8,22 @@ module.exports = {
   'rules': {
     // Prevent missing displayName in a React component definition
     'react/display-name': 0,
+    // Forbid certain propTypes (any, array, object)
+    'react/forbid-prop-types': 0,
```

Comment:
But what about rules that are disabled? What's the reason to include the default config there?

If someone is going to override they'll probably be looking at the docs anyway to see which config they would want to select when enabling.